app/src/main/kotlin/com/x8bit/bitwarden/ui/auth/feature/completeregistration/PasswordStrengthIndicator.kt

```diff
@@ -154,7 +153,10 @@ private fun MinimumCharacterCount(
         }
         Spacer(modifier = Modifier.width(2.dp))
         Text(
-            text = stringResource(BitwardenString.minimum_characters, minimumCharacterCount),
+            text = BitwardenPlurals.minimum_characters.asText(
```

Comment:
The import for `BitwardenPlurals` is missing in several classes, so the project will not compile.